runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpInfoBox.java

```diff
@@ -243,6 +251,36 @@ void update(boolean updated, boolean paused, XpSnapshotSingle xpSnapshotSingle)
 		SwingUtilities.invokeLater(() -> rebuildAsync(updated, paused, xpSnapshotSingle));
 	}
 
+	void toggleHeaderPanel(JPanel progressWrapper)
+	{
+		int progressSkillIconPosition = 0;
+		boolean progressSkillIconState = progressWrapper.getComponent(progressSkillIconPosition).isVisible();
+
+		if (headerPanel.isVisible())
+		{
+			progressWrapper.setBorder(new EmptyBorder(5, 1, 5, 5));
+		}
+		else
+		{
+			progressWrapper.setBorder(DEFAULT_PROGRESS_WRAPPER_BORDER);
+		}
+
+		headerPanel.setVisible(!headerPanel.isVisible());
+		progressWrapper.getComponent(progressSkillIconPosition).setVisible(!progressSkillIconState);
+	}
+
+	private JLabel getSkillIcon(SkillIconManager iconManager, Skill skill, int width, int height, boolean small)
```

Comment:
This method can either be made static or have the SkillIconManager argument removed from it, one or the other.